internal/core/data/init.go

```diff
@@ -73,5 +74,13 @@ func (b *Bootstrap) BootstrapHandler(ctx context.Context, wg *sync.WaitGroup, st
 		},
 	})
 
+	if configuration.MessageQueue.SubscribeEnabled {
```

Comment:
You will need to change the order of the bootstrappers in `main` so that the messaging client is available when this code executes.

Currently:
```
			NewBootstrap(router).BootstrapHandler,
			messaging.BootstrapHandler,
```
Needs to be:
```
			messaging.BootstrapHandler,
			NewBootstrap(router).BootstrapHandler,
```

Reply:
Modified.